### PR TITLE
Fix poincate viz incompatibility with `plotly>=3.0.0`

### LINKED ISSUE #2123: Poincare visualization breaks with plotly 3.0.0
#### Description

poincare_2d_visualization - receives the following error - 
ValueError: 
    Invalid value of type 'builtins.str' received for the 'textposition' property of scatter
        Received value: 'bottom'

#### Steps/Code/Corpus to Reproduce
Example:
```
from gensim.viz.poincare import poincare_2d_visualization
import plotly


vis = poincare_2d_visualization(model, tuples, 'Poincare Plot')
```

#### Versions
Please run the following snippet and paste the output below.
Darwin-17.2.0-x86_64-i386-64bit
Python 3.6.5 |Anaconda, Inc.| (default, Apr 26 2018, 08:42:37) 
[GCC 4.2.1 Compatible Clang 4.0.1 (tags/RELEASE_401/final)]
NumPy 1.14.3
SciPy 1.1.0
gensim 3.5.0
FAST_VERSION 1
Plotly 3.0.0

### PR DESCRIPTION
**Pull request by jenishah**
Fix #2123.

### CODE REVIEW

File: gensim/models/phrases.py [hunk 1]
```diff
@@ -805,7 +805,7 @@ def __init__(self, phrases_model):
         for bigram, score in phrases_model.export_phrases(corpus, self.delimiter, as_tuples=True):
             if bigram in self.phrasegrams:
                 logger.info('Phraser repeat %s', bigram)
-            self.phrasegrams[bigram] = (phrases_model.vocab[self.delimiter.join(bigram)], score)
+            self.phrasegrams[bigram] = score
```
Comment (menshikh-iv) [inline gensim/models/phrases.py]: Not relevant to the original issue, please revert all `phrases.py` file

File: gensim/viz/poincare.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def poincare_2d_visualization(model, tree, figure_title, num_nodes=50, show_node
         mode='markers',
         marker=dict(color='rgb(30, 100, 200)'),
         text=node_labels,
-        textposition='bottom'
+        textposition='bottom center'
```
Comment (menshikh-iv) [inline gensim/viz/poincare.py]: What's about older plotly version? I think PR changes works only for new. This should be compatible with new & old plotly.

Comment (menshikh-iv) [reply gensim/viz/poincare.py]: Checked - it works for 2.7.0 too, great!

Comment (jenishah) [reply gensim/viz/poincare.py]: Thanks! :)

**Comment by menshikh-iv** [comment]
Thanks for contribution @jenishah :+1:

**Pull request merged**